Frameworks/UIKit/UIImage.mm

```diff
@@ -491,8 +491,8 @@ - (void)drawAtPoint:(CGPoint)point blendMode:(CGBlendMode)mode alpha:(float)alph
 
     CGRect pos;
     pos.origin = point;
-    pos.size.width = (img_height / _scale);
-    pos.size.height = (img_width / _scale);
+    pos.size.height = (img_height / _scale);
+    pos.size.width = (img_width / _scale);
```

Comment:
typical think too fast or type too fast.

---
In reply to: [103579177](https://github.com/Microsoft/WinObjC/pull/2117#discussion_r103579177) [](ancestors = 103579177)